app/src/main/kotlin/com/wire/android/ui/home/HomeViewModel.kt

```diff
@@ -40,9 +42,7 @@ class HomeViewModel @Inject constructor(
         private set
 
     init {
-        viewModelScope.launch {
-            launch { loadUserAvatar() }
-        }
+        viewModelScope.launch { loadUserAvatar() }
```

Comment:
basically all the changes in this file could be reverted

Reply:
Indeed.

Just this `viewmodelScope.launch` tiny optimisation, that avoids running a unnecessary `Job` within a `Job`.

I'll revert these.